packages/m2m-gateway/src/services/tenantService.ts

```diff
@@ -85,54 +107,24 @@ export function tenantServiceBuilder(clients: PagoPAInteropBeClients) {
           headers,
         });
 
-      const tenantCertifiedAttributes = tenant.attributes
-        .map((v) => v.certified)
-        .filter(isDefined);
-
-      const tenantCertifiedAttributeIds = tenantCertifiedAttributes.map(
-        (att) => att.id
-      );
-
-      const {
-        data: { results: certifiedAttributes, totalCount },
-      } = await clients.attributeProcessClient.getBulkedAttributes(
-        tenantCertifiedAttributeIds,
-        {
-          headers,
-          queries: {
-            offset,
-            limit,
-          },
-        }
-      );
-
-      const combinedAttributes = zipBy(
-        tenantCertifiedAttributes,
-        certifiedAttributes,
-        ({ id }) => id,
-        ({ id }) => id
-      );
+      const tenantCertifiedAttributes = retrieveCertifiedAttributes(tenant);
```

Comment:
This should be paginated
```suggestion
      const tenantCertifiedAttributes = retrieveCertifiedAttributes(tenant)
        .slice(offset, offset + limit);
```

Reply:
😱 thankss